From 73cc31b90c868526b9c50402debf289d0b43eb30 Mon Sep 17 00:00:00 2001
From: Hoe Hao Cheng <haochengho12907@gmail.com>
Date: Tue, 27 Jun 2023 21:29:57 +0800
Subject: [PATCH] hwdb: fix volume control keys on Lenovo IdeaPad Flex 5
 (14ARE05)

(cherry picked from commit 123c0e24dd3af4c7168ee77dad841cc730b5db60)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 492092e685..ce52a3d47d 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1019,6 +1019,11 @@ evdev:atkbd:dmi:bvn*:bvr*:svnLENOVO:*:pvrIdeaPadFlex514IIL05:*
  KEYBOARD_KEY_ae=!volumedown
  KEYBOARD_KEY_b0=!volumeup
 
+evdev:atkbd:dmi:bvn*:bvr*:svnLENOVO:*:pvrIdeaPadFlex514ARE05:*
+ KEYBOARD_KEY_a0=!mute
+ KEYBOARD_KEY_ae=!volumedown
+ KEYBOARD_KEY_b0=!volumeup
+
 evdev:atkbd:dmi:*:svnLENOVO:*:pvrLenovoYoga300-11IBR:*
  KEYBOARD_KEY_62=unknown  # Touchpad on, also emitted by "Ideapad extra buttons", ignore
  KEYBOARD_KEY_63=unknown  # Touchpad off, also emitted by "Ideapad extra buttons", ignore
